cpu/cortexm_common/vectors_cortexm.c

```diff
@@ -29,7 +29,7 @@
 #include "cpu.h"
 #include "periph_cpu.h"
 #include "kernel_init.h"
-#include "board.h"
+#include "board_generic.h"
```

Comment:
Shouldn't this be included in `board.h` instead?

Reply:
I don't know -- at some point we get to choose.

A nice aspect of separating "that where the functions any board needs to provide" from "that where board-specific macros like LED0_ON are defined" is that it reduces rebuilding, and separates otherwise unrelated concerns.